catalog/app/components/SelectDropdown/SelectDropdown.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react'
+import * as M from '@material-ui/core'
+
+const useStyles = M.makeStyles((t) => ({
+  root: {
+    display: 'inline-block',
+    [t.breakpoints.down('sm')]: {
+      borderRadius: 0,
+      boxShadow: 'none',
+    },
+  },
+  button: {
+    ...t.typography.body1,
+    border: 0,
+    textTransform: 'none',
+  },
+  label: {
+    fontWeight: 600,
+    marginLeft: t.spacing(1),
+  },
+}))
+
+interface ValueBase {
+  toString: () => string
```

Comment:
i meant maybe we dont need `toString()` and `valueOf()` methods here. what do we gain using that instead of plain `{ key, label }` objects?

Reply:
We can use any items as long as they have `toString` and `valueOf`, for example, list of strings or numbers:

```tsx
<SelectDropdown options={[ 'option1', 'option2' ]} value="option1" />
```

```tsx
<SelectDropdown options={[ new Date('2020-01-01'), new Date('2020-01-02') ]} value={new Date('2020-01-01')}/>
```